### ISSUE TITLE
Social buttons

### ISSUE THREAD

**Issue by annapowellsmith** [DemocracyClub/WhoCanIVoteFor#206]
We should add twitter/facebook buttons for easy sharing.

**Comment by annapowellsmith** [comment]
I like http://sharingbuttons.io/ - non JavaScript

**Comment by JoeMitchell** [comment]
As Chris says [here](https://github.com/DemocracyClub/UK-Polling-Stations/issues/344#issuecomment-348457347), need to be careful what exactly we want to share here..

**Comment by chris48s** [comment]
Good point, @joe On WhoCanIVoteFor, it is useful to share deep links. i.e: a share button on a page like

https://whocanivotefor.co.uk/elections/local.city-of-london.2017-12-14/post-LBW:E05009308/portsoken
or
https://whocanivotefor.co.uk/person/5220/ricky-knight

should share the page the button is on (it is useful to be able to link directly to an election or candidate), but https://whocanivotefor.co.uk/elections/mypostcode should not have a share button on it, or it should link to the root domain in that case.
